config/routes/support.rb

```diff
@@ -14,6 +14,7 @@
       end
       resources :courses, only: %i[index edit update]
       resources :locations
+      resources :multiple_locations, path: 'locations/multiple'
```

Comment:
Do we need to feature flag the routes?

Reply:
I think hiding the button will be enough because it's restricted to support users. We can trust that they won't be hacking the URL.